service/constants/constants.js

```diff
@@ -54,7 +54,23 @@ const FileConstants = {
     WORD_TYPES: ['doc', 'docx']
 };
 
+const PostsConstants = {
```

Comment:
It would be more precise to call this something like `PostTypes` because that's all it includes. `PostConstants` implies that is contains all post-related constants, but there are already several in the general `Constants` (like `SYSTEM_MESSAGE_PREFIX`).